src/class/object_users.php

```diff
@@ -516,10 +522,28 @@ function prepareData ($zData = '', $sView = 'list')
                         // Hide the link for phenotypes, because we don't have a phenotypes overview to link to (must be disease-specific).
                         $sOwnes .= (!$sOwnes? '' : ', ') . ($sDataType == 'phenotypes'? $sTitle : '<A href="' . $sDataType . '?search_owned_by_=%3D%22' . rawurlencode(html_entity_decode($zData['name'])) . '%22">' . $sTitle . '</A>');
                     }
+
+                    // Create links to overviews of entries created by user.
+                    $n = $_DB->query('SELECT COUNT(*) FROM ' .
+                        constant('TABLE_' . strtoupper($sDataType)) . ' WHERE created_by = ?',
+                        array($zData['id']))->fetchColumn();
+                    if ($n) {
+                        $nCreates += $n;
+                        $sTitle = $n . ' ' . ($n == 1? substr($sDataType, 0, -1) : $sDataType);
+                        // Hide the link for phenotypes, because we don't have a phenotypes
+                        // overview to link to (must be disease-specific).
+                        $sCreates .= (!$sCreates? '' : ', ') . ($sDataType == 'phenotypes'?
+                                $sTitle : '<A href="' . $sDataType . '?search_created_by=' .
+                                rawurlencode(html_entity_decode($zData['id'])) . '">' .
+                                $sTitle . '</A>');
+                    }
```

Comment:
Perhaps it would be worth merging these two code blocks a bit more? You could save 4 queries to the database, and by adding another loop like `foreach (array('owned_by', 'created_by') as $sField)` you can reduce the code duplication even more. Do you want to go for it?